### DISCUSSION TITLE
Record a prqlc session with asciinema

### DISCUSSION THREAD

**Discussion by vanillajonathan** [PRQL/prql#3656] [Ideas]
Record a `prqlc` session in which you show how to use `prqlc` and some cool stuff with [asciinema](https://asciinema.org/). It could be embedded on the website or in the documentation.

**Comment by richb-hanover** [comment]
What serendipity! I just looked up how to embed an asciinema recording in Markdown... [link](https://asciinema.org/docs/embedding)
